doc/classes/Node.xml

```diff
@@ -578,235 +608,234 @@
 			<param index="1" name="args" type="Array" default="[]" />
 			<param index="2" name="parent_first" type="bool" default="false" />
 			<description>
-				Calls the given method (if present) with the arguments given in [param args] on this node and recursively on all its children. If the [param parent_first] argument is [code]true[/code], the method will be called on the current node first, then on all its children. If [param parent_first] is [code]false[/code], the children will be called first.
+				Calls the given [param method] name, passing [param args] as arguments, on this node and all of its children, recursively.
+				If [param parent_first] is [code]true[/code], the method is called on this node first, then on all of its children. If [code]false[/code], the children's methods are called first.
 			</description>
 		</method>
 		<method name="propagate_notification">
 			<return type="void" />
 			<param index="0" name="what" type="int" />
 			<description>
-				Notifies the current node and all its children recursively by calling [method Object.notification] on all of them.
+				Calls [method Object.notification] with [param what] on this node and all of its children, recursively.
 			</description>
 		</method>
 		<method name="queue_free">
 			<return type="void" />
 			<description>
-				Queues a node for deletion at the end of the current frame. When deleted, all of its child nodes will be deleted as well. This method ensures it's safe to delete the node, contrary to [method Object.free]. Use [method Object.is_queued_for_deletion] to check whether a node will be deleted at the end of the frame.
+				Queues this node to be deleted at the end of the current frame. When deleted, all of its children are deleted as well. Unlike [method Object.free], this method ensures it is safe to delete the node. Use [method Object.is_queued_for_deletion] to check if the node will be deleted at the end of the frame.
 			</description>
 		</method>
 		<method name="remove_child">
 			<return type="void" />
 			<param index="0" name="node" type="Node" />
 			<description>
-				Removes a child node. The node is NOT deleted and must be deleted manually.
-				[b]Note:[/b] This function may set the [member owner] of the removed Node (or its descendants) to be [code]null[/code], if that [member owner] is no longer a parent or ancestor.
+				Removes a child [param node]. The [param node], along with its children, are [b]not[/b] deleted. To delete a node, see [method queue_free].
+				[b]Note:[/b] This method sets the [member owner] of the removed [param node] (or its descendants) to [code]null[/code], if their [member owner] is no longer an ancestor (see [method is_ancestor_of]).
```

Comment:
So it is consistent, and exiting the tree is the cause. It can be interpreted as detaching the branch... Not sure what to make of this one. It my testing, it *does* affect descendant nodes, as well, so long as there's no "reachable" path between node and owner.

Reply:
You may check again, I'm not sure about the issue of this one.

Reply:
You're right about "(or its descendants)".

For setting `owner` to null, I mean it won't happen when no "exit tree" happens. e.g. `foo.remove_child(bar)` when `foo` is not inside tree in the first place. So probably "When this node is inside the tree, this method sets ...".